src/pamet.md

```diff
@@ -24,3 +25,14 @@ anebo jako celé číslo se znaménkem (*signed integer*) **-1** v [dvojkovém d
 
 Čísla v paměti sama o sobě nemají žádný význam, záleží pouze na tom, jak je my, a obzvláště naše programy,
 interpretují a jaké operace nad nimi provádějí.
+
+### Adresování paměti
+Abychom se mohli odkazovat na hodnoty v paměti, tak musíme mít možnost rozlišit jednotlivé buňky od sebe.
+Toho dosáhneme pomocí **adresy**. Paměť je adresována tak, že každá paměťová buňka (každý *byte*)
+má číselnou adresu od 0 do velikosti paměti. Velmi zjednodušeně řečeno, pokud máte RAM paměť o velikosti
+8 GiB (8589934592 "bajtů"), tak můžete adresovat buňky od 0 do 8589934591.
```

Comment:
.. ale aby si pak nemysleli, ze v procesu adresuji bunky naprimo...

Reply:
Good point, dopíšu k tomu něco.

Reply:
OK, mozna by se dala pozdeji napsat rozsirena verze (oddelena pamet procesu), kde je to vysvetleno vice.